src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,18 @@
+$day-bgc: #eee;
+$day-size: 100px;
+$text-size: 30px;
+$gap-size: 1px;
+$cell-hover-bgc:#ffc0cb;
+$transition-duration: 0.5s;
+
+$days: (
+  "mon": 0,
+  "tue": ($day-size + $gap-size),
+  "wed": ($day-size * 2 + $gap-size * 2),
+  "thu": ($day-size * 3 + $gap-size * 3),
+  "fri": ($day-size * 4 + $gap-size * 4),
+  "sat": ($day-size * 5 + $gap-size * 5),
+  "sun": ($day-size * 6 + $gap-size * 6)
```

Comment:
store only number of day here
don't repeat one formula many times, you can specify the formula in css, when you'll set the margins